content/packages/quickstart.md

```diff
@@ -86,13 +86,25 @@ In this guide, you'll create a {% data variables.product.prodname_actions %} wor
             env:
               NODE_AUTH_TOKEN: ${% raw %}{{secrets.GITHUB_TOKEN}}{% endraw %}
     ```
-8. Commit and push your changes to {% data variables.product.prodname_dotcom %}.
+8. Configure publish scope to tell NPM which scope and registry to publish to. This can be acomplished by either
+   - using `.npmrc`
+      ```
+      @YOUR-USERNAME:registry=https://npm.pkg.github.com
+      ```
+   - or using the `publishConfig` key in `package.json`
+      ```json
+      "publishConfig": {
+        "@YOUR-USERNAME:registry": "https://npm.pkg.github.com"
+      }
+      ```
+  
+9. Commit and push your changes to {% data variables.product.prodname_dotcom %}.
     ```shell
     $ git add .github/workflows/release-package.yml
```

Comment:
We need to add either the new `.npmrc` file or the change to `packages.json` here too.

```suggestion
    $ git add .github/workflows/release-package.yml
    # Also add the file you created or edited in the previous step.
    $ git add <em>.npmrc or package.json</em>
```